src/XamlX/IL/RecordingIlEmitter.cs

```diff
@@ -197,11 +197,11 @@ public override string ToString()
             {
                 var i = Instructions[c];
                 sb.AppendFormat("{0000}", c);
-                sb.Append(": ");
+                sb.Append(':').Append(' ');
```

Comment:
CA1834 doesn't prohibit Append of strings longer than one character.

Reply:
You have right, but this change is made to avoiding allocation of ": " in  intern string pool.